### DISCUSSION TITLE
Bug Fixed and improvement new features

### DISCUSSION THREAD

**Discussion by rizwansoaib** [rizwansoaib/whatsapp-monitor#88] [Ideas]
Bug fixed and improvements
group online bug resolved
add favourite number for auto open chats...
csv download icon redefined

<hr /><em>This discussion was created from the release <a href='https://github.com/rizwansoaib/whatsapp-monitor/releases/tag/v2%2C5'>Bug Fixed and improvement new features</a>.</em>

**Comment by RohanSingh1234abcdef** [comment]
Can you share the link to download WhatsApp Monitor v2.6 app for Windows Desktop here?

**Reply by rizwansoaib** [reply]
> Can you share the link to download WhatsApp Monitor v2.6 app for Windows Desktop here?

Desktop App is updated v1.3

**Comment by Abbas141812** [comment]
v.2.6 not working.. please check :(

**Reply by rizwansoaib** [reply]
> v.2.6 not working.. please check :(

Hey please update your browser and whatsapp for everyone it's working personally i have checked also ...

**Reply by Abbas141812** [reply]
downloaded extension via chrome store instead of github..  working now! thanks alot!

**Comment by rizwansoaib** [comment]
Download extension from release if you want to run from source code.

Thanks & regards

On Wed 14 Jul, 2021, 4:52 AM Abbas141812, ***@***.***> wrote:

> downloaded extension via chrome store instead of github.. working now!
> thanks alot!
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rizwansoaib/whatsapp-monitor/discussions/88#discussioncomment-1001338>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AHC2EZFCTUMUTASDK23VJK3TXTDBVANCNFSM5AD6JBFA>
> .
>

-- 
*Disclaimer: *
Messages from 'knit.ac.in <http://knit.ac.in>' mail server 
and any associated files may contain information that is confidential, 
subject to copyright or contains proprietary information. If you are not 
the intended addressee you are hereby notified that any dissemination, 
copying or distribution of these messages, or files associated with these 
messages is prohibited. If you have received the message in error, please 
notify us by replying to the concerned message and deleting it from your 
computer. Please also scan the message to verify that it is virus free. 
Thank you for your time.


This email is governed by the Disclaimer Terms 
of  KNIT.
